### PR TITLE
Make contributing easier

### LINKED ISSUE #239: Add Github issue template
Since we have the daemon and various service providers now (nixos, home-manager, the macos thing…), plus multiple ways to install lorri (self-upgrade, nixpkgs), we should require a bunch of information from users when they open an issue (as easy to get as possible).

We have `lorri info`, so we should add as much diagnostic information as possible to that (so users just have to copy & paste).

### PR DESCRIPTION
**Pull request by curiousleo**
Closes #239.

- [x] Move `CONTRIBUTING.md` to toplevel and add information about the `ci` command
- [x] Create issue template for bug reports
- [x] Create issue template for feature requests
- [x] Create pull request template

---

**"Create issue" screen:**
Screenshot from 2019-12-19 16-03-52

---

**"Create pull request" template preview:**
Screenshot from 2019-12-19 16-01-56

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Metadata**
+Did lorri crash? In this case, please attach the crash report. The crash
+message should point you to the location of the crash report.
+
+In any case, please run `lorri info` inside your project and paste the result.
+
+Please also paste the rust of `uname -a`.
```
Comment (Profpatsch) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Please also paste the result of `uname -a`.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<!--
+Thank you for your contribution!
+
+If this is the first time you are contributing to lorri, please take a look at:
+
+https://github.com/target/lorri/CONTRIBUTING.md
+-->
+
+Closes #ISSUE. <!-- Please replace ISSUE by the issue number this pull request addresses. -->
```
Comment (Profpatsch) [inline .github/PULL_REQUEST_TEMPLATE.md]: Let’s put the comment on the line above so you can jump to the end of the line and delete word.

Comment (curiousleo) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (curiousleo) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thanks!

**Comment by curiousleo** [comment]
> Let’s make sure the user doesn’t have to delete any boilerplate to write an issue, so every comment should be in html comments .

Done, thanks for pointing this out.

**Pull request merged**